src/test/moves/roost.test.ts

```diff
@@ -27,33 +27,205 @@ describe("Moves - Roost", () => {
   beforeEach(() => {
     game = new GameManager(phaserGame);
     game.override.battleType("single");
-    game.override.enemySpecies(Species.STARAPTOR);
-    game.override.enemyAbility(Abilities.INSOMNIA);
+    game.override.enemySpecies(Species.RELICANTH);
     game.override.startingLevel(100);
-    game.override.enemyLevel(100);
-    game.override.moveset([Moves.STOMPING_TANTRUM]);
-    game.override.enemyMoveset([Moves.ROOST, Moves.ROOST, Moves.ROOST, Moves.ROOST]);
+    game.override.enemyLevel(60);
+    game.override.enemyMoveset([Moves.EARTHQUAKE, Moves.EARTHQUAKE, Moves.EARTHQUAKE, Moves.EARTHQUAKE]);
```

Comment:
```suggestion
    game.override.enemyMoveset(Moves.EARTHQUAKE);
```
A recent change to moveset overrides makes this so that the enemy's only move is Earthquake